packages/services/schema/src/lib/__tests__/federation-tag-extraction.spec.ts

```diff
@@ -3,7 +3,7 @@ import {
   applyTagFilterOnSubgraphs,
   applyTagFilterToInaccessibleTransformOnSubgraphSchema,
   type Federation2SubgraphDocumentNodeByTagsFilter,
-} from './federation-tag-extraction';
+} from '../federation-tag-extraction';
```

Comment:
moved this test file into a test directory to match other parts of our code base